embedmaker/embedmaker.py

```diff
@@ -251,34 +250,6 @@ def __init__(self, bot):
             await group.owner.set(ctx.author.id)
             await group.embed.set(serialize_embed(e))
 
-    @_embed.command(name="list")
```

Comment:
Removing this command intentionally?

Reply:
semi-intentionally. I'll need to rewrite the command.